pkg/scd/store/cockroach/store.go

```diff
@@ -127,6 +127,29 @@ func (s *Store) Bootstrap(ctx context.Context) error {
 		INDEX cell_id_idx (cell_id),
 		INDEX operation_id_idx (operation_id)
 	);
+	CREATE TABLE IF NOT EXISTS scd_constraints (
+		id UUID PRIMARY KEY,
+		owner STRING NOT NULL,
+		version INT4 NOT NULL DEFAULT 0,
+		url STRING NOT NULL,
+		altitude_lower REAL,
+		altitude_upper REAL,
+		starts_at TIMESTAMPTZ,
+		ends_at TIMESTAMPTZ,
+		updated_at TIMESTAMPTZ NOT NULL,
+		INDEX owner_idx (owner),
+		INDEX starts_at_idx (starts_at),
+		INDEX ends_at_idx (ends_at),
+		CHECK (starts_at IS NULL OR ends_at IS NULL OR starts_at < ends_at)
+	);
+	CREATE TABLE IF NOT EXISTS scd_cells_constraints (
```

Comment:
i think we should be using the new index/cells format leveraged in rid

Reply:
Done, and yes, that makes things a lot better -- thanks!